docs/self-managed/installation-methods/helm/configure/application-configs.md

```diff
@@ -1,34 +1,54 @@
 ---
 id: application-configs
-sidebar_label: Configure components
-title: Helm charts components configuration
-description: "Learn how to configure individual components in Helm charts."
+sidebar_label: Components
+title: Configure Helm chart components
+description: "Learn how to configure individual Camunda components in Helm charts."
 ---
 
-We moved to using `application.yaml` as the default configuration mechanism, and exposed some options for easier override.
+:::note
+
+- Starting with Helm chart version 8.5, `application.yaml` is the default configuration mechanism.
+- Earlier versions required environment variables for configuration.
+  :::
+
+This page explains how to configure Camunda components in Helm charts. It describes the shift from environment variables to `application.yaml`, and shows how to apply configuration options and custom files.
+
+## Prerequisites
+
+- A deployed Camunda Helm chart release.
+- Access to the `values.yaml` file.
+- Basic understanding of Spring Boot configuration (`application.yaml` or `application.properties`).
+
+## Configuration
 
-Before, we set many environment variables as uppercase variants of the Spring option names. This can be confusing as most of our documentation is written with the Spring `application.properties`
-or `application.yaml` names in mind. We would also run into problems where components released a feature, but the Helm chart would not expose the option under a key.
+### Parameters
 
-## Process with environment variables (before 8.5)
+| Key                                  | Type   | Description                                                                                                             |
+| ------------------------------------ | ------ | ----------------------------------------------------------------------------------------------------------------------- |
+| `<componentName>.configuration`      | string | Full application configuration content (for example, the contents of `application.yaml`).                               |
+| `<componentName>.extraConfiguration` | map    | Additional configuration files. Keys are filenames, values are file contents. Mounted into the container at `./config`. |
 
-Previously, if you wanted to customize the application configurations you would need to know how [Spring would read properties](https://docs.spring.io/spring-boot/docs/1.5.6.RELEASE/reference/html/boot-features-external-config.html).
+### Environment variables (before 8.5)
```

Comment:
Same as top - I will remove this section as 8.5 is not supported anymore after the 8.8 release.